### DISCUSSION TITLE
Vuex ORM like this.$repo

### DISCUSSION THREAD

**Discussion by ghost** [CodeDredd/pinia-orm#686] [Q&A] [answered]
Hello. How can I use pinia-orm like in vuex-orm with `store.$repo` or mapState from pinia with for example
`  computed: {
    ...mapState({
      appContextState: (state) => state.entities.appContext,
    }),
  },`

**Comment by CodeDredd** [comment, marked as answer]
Hi. In pinia-orm there is no `store.$repo` although i see i still have this in the documentaion. Need to remove it.
If you are using options api then you can use the `mapRepos` helper.
Look there: https://pinia-orm.codedredd.de/guide/repository/getting-started#map-repositories-helper

**Reply by ghost** [reply]
Thank you
